spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'As a visitor' do
+  describe "When I visit the root path and search for 'sweet potatoes'" do
+    before :each do
+      visit "/"
+      fill_in (:q), with: 'sweet potatoes'
+      click_button 'Search'
+    end
+
+    it 'the page should direct to foods index' do
+      expect(current_path).to eq(foods_path)
+    end
+
+    it 'the page should display total number of items returned in search' do
+    end
+
+    it 'the page should display top ten most relevant foods related to ingredient' do
+    end
+
+    it 'each food on list should list GTIN/UPC code, description, Brand Owner, and ingredients' do
+    end
+  end
+end
```

Comment:
For testing, once you have it all refactored like you do now, I like to think of the following: 

PORO test: 
- tests any attributes as well as any methods (you didn't have any) that are in the poro)
- write a test that takes in the data that you're assuming you'll pass to it (json object with food data)
- test that when you call methods (attributes) on that object, that you get what you'd expect back 

Facade test: 
- write a test that tests the return value of the facade -- in your case, you want your facade method to return food objects, so test that when the method gets called, that the objects you get returned are all food objects 

Service test: 
- this one feels a bit vague, but you really just need to test the structure you're expecting to get back 
- usually I'll hit the endpoint using postman, and take a look at that response to get an idea for what I can test 
- you'll test the structure (key's and values) of the json response you're expecting the service call to return to you.